// ngx-gauge/esm5/gauge/gauge-directives.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Directive } from "@angular/core";
var NgxGaugeAppend = /** @class */ (function () {
    function NgxGaugeAppend() {
    }
    NgxGaugeAppend.decorators = [
        { type: Directive, args: [{
                    selector: "ngx-gauge-append",
                    exportAs: "ngxGaugeAppend"
                },] }
    ];
    return NgxGaugeAppend;
}());
export { NgxGaugeAppend };
var NgxGaugePrepend = /** @class */ (function () {
    function NgxGaugePrepend() {
    }
    NgxGaugePrepend.decorators = [
        { type: Directive, args: [{
                    selector: "ngx-gauge-prepend",
                    exportAs: "ngxGaugePrepend"
                },] }
    ];
    return NgxGaugePrepend;
}());
export { NgxGaugePrepend };
var NgxGaugeValue = /** @class */ (function () {
    function NgxGaugeValue() {
    }
    NgxGaugeValue.decorators = [
        { type: Directive, args: [{
                    selector: "ngx-gauge-value",
                    exportAs: "ngxGaugeValue"
                },] }
    ];
    return NgxGaugeValue;
}());
export { NgxGaugeValue };
var NgxGaugeLabel = /** @class */ (function () {
    function NgxGaugeLabel() {
    }
    NgxGaugeLabel.decorators = [
        { type: Directive, args: [{
                    selector: "ngx-gauge-label",
                    exportAs: "ngxGaugeLabel"
                },] }
    ];
    return NgxGaugeLabel;
}());
export { NgxGaugeLabel };
